spec/support/event_subscriber.rb

```diff
@@ -53,13 +53,26 @@ def failed(event)
       end
     end
 
+    def select_cmap_events(cls)
```

Comment:
This doesn't seem to have anything specific to CMAP in it; maybe it should jut be named `select_succeeded_events`?

Reply:
The purpose of this method is to select cmap events. As CMAP implementation is incomplete, the implementation of this method is subject to change as well.

Reply:
The current name of the method does not reflect what it does very well, given that it works fine for any event class, not just CMAP ones. If future changes change the semantics of this method, then the name can be changed then to reflect it. This is code only used internally in the tests, so there's no issue with changing the name of it later if it needs to be changed to become CMAP-specific later (which doesn't seem like it would definitely have to be the case).

Reply:
This method exists to select cmap events, and that is what it does. The fact that it may be used for another purpose does not necessarily make it appropriate for other purposes. If in the future this method ends up being used for something else, its name and/or its implementation may be adjusted. As you pointed out, this is easy to do.

Reply:
I still think it's pretty silly to arbitrarily claim that the method only works for CMAP events when it very clearly doesn't, but since this isn't a public API, it's not worth blocking these changes on, so I'll resolve this conversation.